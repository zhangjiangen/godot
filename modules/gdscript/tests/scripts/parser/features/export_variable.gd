@export var example = 99
@export_range(0, 100) var example_range = 100
@export_range(0, 100, 1) var example_range_step = 101
@export_range(0, 100, 1, "or_greater") var example_range_step_or_greater = 102


func test():
	print(example)
	print(example_range)
	print(example_range_step)
	print(example_range_step_or_greater)
