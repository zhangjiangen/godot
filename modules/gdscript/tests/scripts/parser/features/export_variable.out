GDTEST_OK
99
100
101
102
